estate/models/estate_property.py

```diff
@@ -0,0 +1,144 @@
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
```

Comment:
```suggestion
from datetime import datetime
from dateutil.relativedelta import relativedelta

from odoo import api, fields, models
from odoo.exceptions import UserError
```
Generally we first make external imports and then we import from odoo in alphabetical order.
Need to adapt in all files